.github/workflows/test-install.yml

```diff
@@ -183,6 +183,7 @@ jobs:
     - name: Run test suite
       env:
         AIIDA_TEST_BACKEND: ${{ matrix.backend }}
+        PYTEST_ADDOPTS: "-v"
```

Comment:
perhaps this should be added in `.github/workflows/tests.sh` instead, with the other `PYTEST_ADDOPTS` lines?
I'm not too bothered either way though

Reply:
I opted to put them here in case we don't want to increase verbosity for all workflows, but it's true, might make sense to increase verbosity in all test runs.